src/spd-notification-worker/src/main/java/ca/bc/gov/open/pssg/rsbc/spd/notification/worker/OutputNotificationConsumer.java

```diff
@@ -14,13 +31,138 @@
 @Component
 public class OutputNotificationConsumer {
 
-    Logger logger = LoggerFactory.getLogger(this.getClass());
+    private static final String IMAGE_EXTENSION = "PDF";
+    private static final int SUCCESS_CODE = 0;
+    private static final String DPS_FILE_ID_KEY = "dps.fileId";
+    private static final String DPS_BUSINESS_AREA_CD_KEY = "dps.businessAreaCd";
+
+    private final FileService fileService;
+    private final SftpProperties sftpProperties;
+    private final DocumentService documentService;
+    private final JAXBContext kofaxOutputMetadataContext;
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public OutputNotificationConsumer(FileService fileService,
+                                      SftpProperties sftpProperties,
+                                      DocumentService documentService,
+                                      @Qualifier("kofaxOutputMetadataContext") JAXBContext kofaxOutputMetadataContext) {
+        this.fileService = fileService;
+        this.sftpProperties = sftpProperties;
+        this.documentService = documentService;
+        this.kofaxOutputMetadataContext = kofaxOutputMetadataContext;
+    }
 
     @RabbitListener(queues = Keys.CRRP_QUEUE_NAME)
     public void receiveMessage(OutputNotificationMessage message) {
 
-        logger.info("received message for {}", message.getBusinessAreaCd());
+        logger.info("received new {}", message);
+
+        MDC.put(DPS_FILE_ID_KEY, message.getFileId());
+        MDC.put(DPS_BUSINESS_AREA_CD_KEY, message.getBusinessAreaCd());
+
+        FileInfo fileInfo = new FileInfo(message.getFileId(), IMAGE_EXTENSION, sftpProperties.getRemoteLocation());
+        logger.info("fileInfo {}", fileInfo.toString());
+
+        try {
+
+            logger.debug("attempting to download file [{}]", fileInfo.getMetaDataReleaseFileName());
+            String metadata = getMetadata(fileInfo);
+            logger.info("successfully downloaded file [{}]", fileInfo.getMetaDataReleaseFileName());
+
+            Data parsedData = unmarshallMetadataXml(metadata);
+            Data.DocumentData documentData = parsedData.getDocumentData();
 
+            DpsDocumentRequestBody documentRequestBody = new DpsDocumentRequestBody(sftpProperties.getHost(),
+                    fileInfo.getImageReleaseFileName());
+            logger.info("dpsDocumentRequestBody: {}", documentRequestBody.toString());
+
+            DpsDocumentResponse documentResponse = documentService.dpsDocument(documentRequestBody);
+            logger.info("dpsDocumentResponse: {}", documentResponse.toString());
```

Comment:
Can you remove this log, it would put the body in the logs and replace with success on the operation